### PR TITLE
VI: use double height to calc AR in DS mode

### PR DESCRIPTION
**Pull request by booto**
Attempt to get doublestrike video modes to display less "squished"

### CODE REVIEW

**Comment by phire** [comment]
240Int mode is squished.

**Comment by phire** [comment]
Ok, that fixes 240int mode

File: Source/Core/Core/HW/VideoInterface.cpp [hunk 1]
```diff
@@ -469,10 +469,15 @@ static u32 GetTicksPerOddField()
...
 {
-	u32 multiplier = static_cast<u32>(m_PictureConfiguration.STD / m_PictureConfiguration.WPL);
-	int height = (multiplier * m_VerticalTimingRegister.ACV);
-	int width = ((2 * m_HTiming0.HLW) - (m_HTiming0.HLW - m_HTiming1.HBS640)
-		- m_HTiming1.HBE640);
+	u32 multiplier = m_DisplayControlRegister.NIN?1:2;
+
+	// if this is doublestrike mode, each field has an even amount of halflines and
+	// it's effectively double the height
+	if ((m_VerticalTimingRegister.EQU<10) && ((GetHalfLinesPerEvenField() & 1) == 0) && ((GetHalfLinesPerOddField() & 1) == 0)) {
```
Comment (lioncash) [inline Source/Core/Core/HW/VideoInterface.cpp]: brace on new line, also spaces on both sides of `<`

File: Source/Core/Core/HW/VideoInterface.cpp [hunk 2]
```diff
@@ -469,10 +469,15 @@ static u32 GetTicksPerOddField()
...
+		multiplier = 2;
+	}
+	u32 height = (multiplier * m_VerticalTimingRegister.ACV);
+	u32 width = 2 * m_HTiming1.HBS640 -  m_HTiming1.HBE640;
```
Comment (lioncash) [inline Source/Core/Core/HW/VideoInterface.cpp]: one space too many after the `-`

File: Source/Core/Core/HW/VideoInterface.cpp [hunk 3]
```diff
@@ -469,10 +469,15 @@ static u32 GetTicksPerOddField()
 
 float GetAspectRatio(bool wide)
 {
-	u32 multiplier = static_cast<u32>(m_PictureConfiguration.STD / m_PictureConfiguration.WPL);
-	int height = (multiplier * m_VerticalTimingRegister.ACV);
-	int width = ((2 * m_HTiming0.HLW) - (m_HTiming0.HLW - m_HTiming1.HBS640)
-		- m_HTiming1.HBE640);
+	u32 multiplier = m_DisplayControlRegister.NIN?1:2;
```
Comment (JosJuice) [inline Source/Core/Core/HW/VideoInterface.cpp]: Spaces around `?` and `:`

**Comment by booto** [comment]
Dolphin's now in feature freeze and this addresses on of the ["Milestone: Current" issues](https://bugs.dolphin-emu.org/issues/9035). @phire, what specific problems do you have with this PR, and if this PR does need to be rejected, what is the ETA for an alternative?

**Comment by booto** [comment]
Superseded by #3472

**Pull request closed without merging**